jgrapht-core/src/test/java/org/jgrapht/traverse/LexBreadthFirstIteratorTest.java

```diff
@@ -31,41 +31,38 @@
  *
  * @author Timofey Chudakov
  */
-public class LexBreadthFirstIteratorTest
-{
+public class LexBreadthFirstIteratorTest {
```

Comment:
When reformatting, make sure your editor is set to use our conventions in order to avoid spurious changes.

https://github.com/jgrapht/jgrapht/wiki/How-to-setup-your-development-environment-for-JGraphT#developing-using-eclipse